index.html

```diff
@@ -92,26 +92,15 @@ <h1>Women in Coding</h1>
         <section class="row">
             <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                 <p>Our mission is to provide resources and opportunities for those interested in coding and design in the local Syracuse
-                    community. Although this group is geared towards women, anyone is welcome to attend our classes.</p>
-                <p>We currently host monthly classes and group programming sessions where participants can learn about a
-                    new topic and practice it with guidance from mentors.
-                </p>
+                    community. Workshops and meetups are organized and hosted by local women in tech, but all are welcome to attend our classes.</p>
                 <hr class="line" />
                 <h2>JOIN US!</h2>
-                <p>Our monthly workshops are meant for everyone from the code newbie to the experienced developer. Each
-                    meetup will have 1 or 2 mentors who can provide support and answer questions. Bring your laptop and
-                    an open mind. 
+                <p>Our monthly workshops are meant for everyone from the code newbie to the experienced developer.
                 <ul>
-                    <li>Workshops are currently held on virtually on Zoom, usually on the second Saturday of each month from 10am to 12pm.</p>
+                    <li>Workshops are usually held virtually on Zoom, usually on the second Saturday of each month from 10am to 12pm.</p>
```

Comment:
I would take the second usually out in that sentence. "Workshops are usually held virtually on Zoom on the second Saturday of each month from 10 AM to 12 PM EST." reads fine to me.